aries-site/src/themes/aries.js

```diff
@@ -3,6 +3,12 @@ import { deepMerge } from 'grommet/utils';
 
 export const aries = deepMerge(hpe, {
   defaultMode: 'dark',
+  spinner: {
+    size: {
+      small: '36px',
```

Comment:
In the designs I'm seeing 48px in some places, 36px in others, and 32px in others. Can we align with @vavalos5 on the expected dimensions before overriding grommet default?